.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -32,8 +34,6 @@ Links and Details can be found in the samples WG Teams Files.
 - [ ] Adhere to readme template 
 - [ ] Ensure/create CI test configurations for sample (ciTests field) https://github.com/oneapi-src/oneAPI-samples/wiki/sample-json-ci-test-object
 - [ ] Enforce format via clang-format config file
-- [ ] Compile code using compiler flags and fix anything detected "enable /Wall -Wformat-security -Werror=format-security"
-- [ ] Review Sample with Domain Expert: <insert NameHere>
```

Comment:
is there a reason you are removing this?

Reply:
Removed accidentally, re-inserted now.